src/views/Home/hooks/useFetchLotteryForPromos.ts

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react'
+import { fetchCurrentLotteryIdAndMaxBuy, fetchLottery } from 'state/lottery/helpers'
+import useRefresh from 'hooks/useRefresh'
+import useIntersectionObserver from 'hooks/useIntersectionObserver'
+
+const useFetchLotteryForPromos = () => {
+  const { slowRefresh } = useRefresh()
+  const { observerRef, isIntersecting } = useIntersectionObserver()
```

Comment:
is this needed here? 🤔

Reply:
This should be removed, and should be used in compenents if it is needed (in this case i don't think we need it since in mobile and desktop banner is always visible when page is loaded)

Reply:
🗑️